### DISCUSSION TITLE
Need some help getting fumadocs-mdx v10 working in turbo monorepo

### DISCUSSION THREAD

**Discussion by idebeijer** [fuma-nama/fumadocs#804] [Q&A]
I am trying to upgrade fumadocs-mdx to v10, and adding the script `"postinstall": "pnpm dlx fumadocs-mdx"` to one of my apps in my turborepo successfully creates the `.source` folder within that app. But the postinstall script then fails. It looks like it tries to find `.source` from the root node_modules rather than the node_modules of the turborepo app, is that the desired behavior?

```bash
apps/docs postinstall$ pnpm dlx fumadocs-mdx
│ node:internal/modules/esm/resolve:265
│     throw new ERR_MODULE_NOT_FOUND(
│           ^
│ Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/path/to/project/monorepo/node_modules/.sou…
│     at finalizeResolution (node:internal/modules/esm/resolve:265:11)
│     at moduleResolve (node:internal/modules/esm/resolve:933:10)
│     at defaultResolve (node:internal/modules/esm/resolve:1169:11)
│     at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)
│     at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)
│     at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:227:38)
│     at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:87:39)
│     at link (node:internal/modules/esm/module_job:86:36) {
│   code: 'ERR_MODULE_NOT_FOUND',
│   url: 'file:///path/to/project/monorepo/node_modules/.source'
│ }
```

Is this misconfiguration on my part, or doesn't fumadocs-mdx v10 support using it in a turborepo?

**Comment by fuma-nama** [comment]
You don't need dlx for postinstall command

**Reply by idebeijer** [reply]
Nevermind, I was doing something rather stupid. I had an enum defined in source.ts where the loaders are, because before v10 that is where the frontmatter schema could be extended. But now in v10 the frontmatter schema needs to be extended in `source.config.ts` and since I hadn't moved the enum out of source.ts it was just imported from there to the new `source.config.ts`, moving the enum out of source.ts and having no imports from source.ts in source.config.ts fixed my issue.
